ONBOARD.md

```diff
@@ -0,0 +1,183 @@
+This document describes how to add new CI jobs (utilizing ci-operator) for
+OpenShift components to the OpenShift CI system. In general, two steps are
+needed:
+
+1. [Prepare configuration file](#prepare-configuration-for-component-repo),
+   describing how ci-operator should build and test your component.
+2. [Add new jobs to Prow](#add-prow-jobs), hooking ci-operator to important
+   GitHub events for your repository.
+
+This walkthrough covers a simple use case of hooking up a job running source
+code level tests (unit, performance etc.) on pull requests on a component
+repository. Although ci-operator can perform more sophisticated test steps than
+simply running unit tests, these use cases are beyond the scope of this doc.
+
+## Prepare configuration for component repo
+
+The JSON configuration file describes how to build different images in a
+testing pipeline for your repository. ci-operator has different *”targets”*
+that describe the goal images to build, and later targets build on successfully
+built earlier targets. You will probably want to write and test your config file
```

Comment:
What does "test your config file" mean? How do you know it "works"?

Reply:
It works once you can locally run `ci-operator` without `--target` and with `--git-ref` and it builds all targets successfully, provided the tests actually pass for that ref. I'll try to clarify.